### DISCUSSION TITLE
a_streamer.json exclude parameters

### DISCUSSION THREAD

**Discussion by 13Quavo** [RakambdaOrg/ChannelPointsMiner#239] [General]
Hello,

I am currently using 3 accounts to mine streamers. Is there any way that a certain account exclude certain streamer to mine using some parameter without unfollowing them?
Thank you and I look forward to your response!

**Comment by Rakambda** [comment]
Hmm, currently there isn't a "nice way" to basically exclude streamers.

You can either :

* not enable follows in the bot, and define everything in the streamers folder. That's personally what I do as I can configure the bot however I want independantly of whatever happens on twitch.
* override the config for that streamer and give it a constant score that is very low (like -99999). That won't stop him from being mined, but he'll be the last option picked.
* (that's more a joke but I still put it 😏) get banned from the chat

**Reply by Rakambda** [reply]
Though what could be done is have something similar to the account, with an "enabled" flag.

This way you could could create a streamer.json with something like 

```json
{"enabled":false}
```

And it'd exclude him from being mined and from joining chat.

But that's have to be looked into as it may not be that straight forward in the code to "override" the behavior of a defined streamer.

---

EDIT: I don't know how you use the app (through a git download or with docker), but it you're at ease with it you can try the `feature/exclude` branch (#240)

**Reply by 13Quavo** [reply]
Thank you ❤️
